examples/with-remix-thirdpartyemailpassword/app/lib/superTokensTypes.ts

```diff
@@ -0,0 +1,48 @@
+import { SessionContainerInterface } from "supertokens-node/lib/build/recipe/session/types";
+
+export type PartialRemixRequest = {
```

Comment:
do we really need this type? Think of  a way in which it's not needed

Reply:
Refactored so this type isn't needed.